go.mod

```diff
@@ -5,29 +5,25 @@ go 1.17
 require (
 	github.com/ThalesIgnite/crypto11 v1.2.5
 	github.com/coreos/go-systemd/v22 v22.5.0
-	github.com/ethereum/go-ethereum v1.10.22
 	github.com/google/uuid v1.2.0
 	github.com/miekg/pkcs11 v1.0.3-0.20190429190417-a667d056470f
 	github.com/pelletier/go-toml v1.8.0
 	github.com/stretchr/testify v1.7.2
+	github.com/umbracle/ecies v0.0.0-20190516151710-4bface869a50
```

Comment:
It looks like the `umbracle/ecies` is fork from `https://github.com/obscuren/ecies`, why not use the original one?

Reply:
The license is BSD-3.

Reply:
> This implementation is direct fork of Kylom's implementation. I claim no authorship over this code apart from some minor modifications. Please be aware this code **has not yet been reviewed**. 

So do we know if the other code has been reviewed? 😊

Reply:
There is a dozen of forks on the `obscuren/ecies`, which itself is a fork from `go-ethereum`, which is a work by Google engineer granted to `go-ethereum` under the Golang-compatible Google's license.

But only the `umbracle/ecies` seemed like maintaned at least 3 years ago and have some fixes which make it work with the recent Golang versions. All other forks, as well, as the original are practically dead ends for many years.

It looks like everybody [in need of ECIES and using Golang] cares not about the license or the import of universe. So all well maintained OS projects I've seen so far simply use the `go-ethereum`'s implementation of ECIES.